src/main/java/com/saralein/server/response/DirectoryResponse.java

```diff
@@ -16,7 +21,7 @@ private String createHeader() {
     }
 
     private String createBody() {
-        FileHelper fileHelper = new FileHelper("public");
+//        SysFileHelper fileHelper = new SysFileHelper("public");
```

Comment:
I'm not sure there's much value in this comment

Reply:
Definitely meant to remove that.  Will do so.  Thank you.